page-blueprints/weather-dashboard-openweathermap/blueprint.yaml

```diff
@@ -214,6 +213,89 @@ card:
+                          content: $Risks$ # <- translate the button name
+                          tap_action:
+                            action: navigate
+                            navigation_path: '#risks'
+                          hold_action:
+                            action: none
+                    - type: custom:weather-card
+                      style: |
+                        ha-card {
+                          border-radius: 10px;
+                          padding-bottom: 10px;
+                          background-color: var(--dwains-theme-primary)
+                        }
+                        :host {
+                          --paper-item-icon-color: var(--dwains-theme-accent) !important;
+                        }
+                        .card-header {
+                          padding: 5px 16px;
+                          font-size: 15px;
+                          font-weight: 700 !important;
+                        }
+                        #states {
+                          padding-top: 0px !important;
+                          padding-bottom: 0px !important;
+                        }
+                        .secondary {
+                          color: darkgray !important;
+                          margin-left: 2px !important;
+                        }
+                      entity: $weather_entity$
+                      current: true
+                      details: true
+                      forecast: true
+                      hourly_forecast: false
+                      number_of_forecasts: '5'
+                    - type: entities
+                      title: $Stats$
+                      style: |
+                        ha-card {
+                          border-radius: 10px;
+                          padding-bottom: 10px;
+                          background-color: var(--dwains-theme-primary)
+                        }
+                        :host {
+                          --paper-item-icon-color: var(--dwains-theme-accent) !important;
+                        }
+                        .card-header {
+                          padding: 5px 16px;
+                          font-size: 15px;
+                          font-weight: 700 !important;
+                        }
+                        #states {
+                          padding-top: 0px !important;
+                          padding-bottom: 0px !important;
+                        }
+                        .secondary {
+                          color: darkgray !important;
+                          margin-left: 2px !important;
+                        }
+                      entities:
+                        - entity: $weather_entity_feels_like$
+                          name: $Feels_Like$
+                        - entity: $wind_speed$
+                          name: $Wind_speed$
+                        - entity: $weather_entity_humidity$
+                          name: $Humditiy$
```

Comment:
There's a spelling mistake here which causes the label not to load. r/Humditiy/Humidity/